components/multi-choice/src/test.js

```diff
@@ -1,38 +1,33 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
 import { mount } from 'enzyme';
 
 import MultiChoice from './';
 
 describe(MultiChoice, () => {
-  const example = 'example';
-  const meta = {
-    touched: true,
-    error: example,
-  };
-  const wrapper = (
-    <MultiChoice meta={meta} hint={example} label={example}>
-      {example}
-    </MultiChoice>);
+  it('renders with a label', () => {
+    const wrapper = mount(<MultiChoice label="Label">Example</MultiChoice>);
```

Comment:
Ideally the different fixtures used by tests should be in `fixtures.js` and used by both `stories.js` and `test.js` to ensure that what we have covered by snapshot tests also appears in our visual regression tests.

Reply:
would these new snapshot tests be useful to have on storybook examples/chromatic?
If so, please add them via fixtures.js
If not, why not?

Reply:
👍 I'll take a look at this and see if I can break out anything shared.